docs/sql-migration-guide.md

```diff
@@ -256,6 +256,52 @@ license: |
 
   - Since Spark 3.0, the unary arithmetic operator plus(`+`) only accepts string, numeric and interval type values as inputs. Besides, `+` with a integral string representation will be coerced to double value, e.g. `+'1'` results `1.0`. In Spark version 2.4 and earlier, this operator is ignored. There is no type checking for it, thus, all type values with a `+` prefix are valid, e.g. `+ array(1, 2)` is valid and results `[1, 2]`. Besides, there is no type coercion for it at all, e.g. in Spark 2.4, the result of `+'1'` is string `1`.
 
+  - Since Spark 3.0, the parameter(first or second) to array_contains function is implicitly promoted to the wider type parameter.
```

Comment:
@maropu @srowen do we still need a migration guide? It looks an obvious bug to me that we forget to do type coercion for decimal types. And I don't think a user would expect Spark to fail `array_contains` with compatible decimal types.

Reply:
Ah, I see. The current latest fix looks reasonable. This fix is not a behaviour change but a bug fix.

Reply:
Thanks @cloud-fan @maropu 
I will revert these changes from migration guide.